### ISSUE TITLE
Should requests be logged in non dev environments?

### ISSUE THREAD

**Issue by bengourley** [serby/ctrl#129]
The express logger might contain some useful debugging info. Currently we only log requests in development.

**Comment by serby** [comment]
Yes. I took it out before. Happy to have it back if it well considered.

**Comment by bengourley** [comment]
Could be handy, but on the other hand it would create a lot of noise if you're looking for something else in the log file (plus the issue of storage amount and potential latency writing to a large file)
